### PR TITLE
Use pip-tools to pin dependencies of pip-tools

### PR DESCRIPTION
**Pull request by vasilty**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -27,8 +29,9 @@ python-dateutil==2.7.3    # via croniter
 pytz==2019.3              # via -r requirements.in, redis-tasks
 redis-tasks==0.0.8        # via -r requirements.in
 redis==2.10.6             # via redis-tasks
-six==1.11.0               # via more-itertools, packaging, python-dateutil
+six==1.11.0               # via more-itertools, packaging, pip-tools, python-dateutil
 testfixtures==6.14.0      # via flake8-isort
 toml==0.10.0              # via isort
 wcwidth==0.1.9            # via pytest
 werkzeug==1.0.0           # via -r requirements.in
+zipp==3.1.0               # via importlib-metadata
```
Comment (vasilty) [inline requirements.txt]: @i-trofimtschuk Look, infamous `zipp` and `importlib-metadata` appeared. These get removed by dependabot

**Review by i-trofimtschuk — APPROVED** [review]

**Comment by i-trofimtschuk** [comment]
@vasilty great! shall we merge it?

**Review by kaloyanm — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (i-trofimtschuk) [reply requirements.txt]: @vasilty could this also be fixed by .python-version  ?

Comment (vasilty) [reply requirements.txt]: Yes, I hope so. But I still would like to let pip tools manage pip tools dependencies as some are commaon, like click, and we would get issues if we overwrite one with an incompatible version if installed directly

**Pull request merged**